priv/templates/extended_bin

```diff
@@ -694,6 +694,14 @@ case "$1" in
                 START_OPTION="console_boot"
                 HEART_OPTION="daemon_boot"
                 ;;
+            start)
+                # TODO, add here the right annoying message asking users to consider
+                # instead using systemd or some such other init system
+                echo "'start' has been deprecated and will be removed in the short-term, please consider <link-to-doc> for alternatives"
```

Comment:
Should add in that the user should use `daemon` to get the same result as `start` and also to check the link to docs for why you should really be using foreground and an init tool. But don't have to mention foreground if we get a link in the docs for this, but do need it to say `use daemon instead`.

Same for `attach`, it should say `attach is deprecated, use daemon_attach ...`

Reply:
ok, i'll rephrase this to not require the doc link just now, we can open a new PR to add it later